src/commands/git.rs

```diff
@@ -452,10 +455,18 @@ fn cmd_git_clone(
 fn do_git_clone(
     ui: &mut Ui,
     command: &CommandHelper,
+    colocate: bool,
     source: &str,
     wc_path: &Path,
 ) -> Result<(WorkspaceCommandHelper, Option<String>), CommandError> {
-    let (workspace, repo) = Workspace::init_internal_git(command.settings(), wc_path)?;
+    let (workspace, repo) = if colocate {
+        let git_path = &wc_path.join(".git");
+        // Need to clone the git repo first.
```

Comment:
I find this comment confusing. Is it left over from an earlier version, because this block of code doesn't do any cloning. I would just delete the comment because it seems pretty obvious what it does.